continuous-benchmarking/database-operations/write-results-to-db/lambda_function.py

```diff
@@ -0,0 +1,81 @@
+import json
+import logging
+import os
+import time
+import uuid
+
+import boto3
+dynamodb = boto3.resource('dynamodb')
+
+def lambda_handler(event, context):
+    
+    logger = logging.getLogger()
+    logger.setLevel(logging.INFO)
+    logger.info("Request: %s", event)
+	
+    data = json.loads(str(event['body']).replace("'",'"'))
+    response = ''
+    
+    # Partition key - experiment_type (Mandatory)
+    # Sort key - date (Mandatory)
+    
+    experiment_type = data.get('experiment_type')
+    date = data.get('date')
+    
+    if not (experiment_type and date):
+        response = 'required exp or date'
+        logging.error("Experiment type is required.")
+        raise Exception("Couldn't write data.")
+        
+    timestamp = str(time.time())
+
+    table = dynamodb.Table('continous_results')
+    
+    # Extract params from body
+    subtype = data.get('subtype')
```

Comment:
why not inline the `data.get('xx')` in the entry below because these variables are used only once